documentation/conf.asciidoc

```diff
@@ -23,7 +23,7 @@ This folder contains configurations for your IDE:
 The `.m2` folder is used for configurations of link:mvn.asciidoc[maven]. It contains the local `repository` folder used as cache for artifacts downloaded and installed by maven (see also https://maven.apache.org/guides/introduction/introduction-to-repositories.html[maven repositories]).
 Further, there are two configuration files for maven:
 
-* https://maven.apache.org/settings.html[settings.xml] initialized from a template from your `devonfw-ide` xref:settings[]. You may customize this to your needs (configuring HTTP proxies, credentials, or other user-specific settings).
+* https://maven.apache.org/settings.html[settings.xml] initialized from a template from your `devonfw-ide` xref:settings[]. You may customize this to your needs (configuring HTTP proxies, credentials, or other user-specific settings). Secrets can be specified as `$[«variable.name»]` and will be prompted, encrypted and replaced automatically during the setup (unless in batch mode).
```

Comment:
Could you provide a link here to the section which explains what batch mode is?

Reply:
OK, "batch mode" was not documented in asciidoc before either. Added it and linked here.